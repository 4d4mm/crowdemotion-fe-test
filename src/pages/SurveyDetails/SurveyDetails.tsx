import * as React from "react";
import Layout from "../../templates/Layout/Layout";

const SurveyDetails = ({}) => <div>SurveyDetails</div>;
export default SurveyDetails;
