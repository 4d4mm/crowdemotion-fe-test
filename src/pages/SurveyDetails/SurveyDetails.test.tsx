import * as React from "react";
import { mount } from "enzyme";
import { MemoryRouter } from "react-router-dom";
import SurveyDetails from "./SurveyDetails";

describe("SurveyDetails", () => {
  it("renders SurveyDetails", () => {
    const surveyDetails = mount(<MemoryRouter initialEntries={[{pathname:'/', key:'test'}]}><SurveyDetails /></MemoryRouter>);
    expect(surveyDetails).toMatchSnapshot();
  });
});
