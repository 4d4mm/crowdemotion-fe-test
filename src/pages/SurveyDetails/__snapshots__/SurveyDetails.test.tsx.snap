// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurveyDetails renders SurveyDetails 1`] = `
<MemoryRouter
  initialEntries={
    Array [
      Object {
        "key": "test",
        "pathname": "/",
      },
    ]
  }
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "canGo": [Function],
        "createHref": [Function],
        "entries": Array [
          Object {
            "hash": "",
            "key": "test",
            "pathname": "/",
            "search": "",
          },
        ],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "index": 0,
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "key": "test",
          "pathname": "/",
          "search": "",
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <SurveyDetails>
      <div>
        SurveyDetails
      </div>
    </SurveyDetails>
  </Router>
</MemoryRouter>
`;
